src/offices/_single.html

```diff
@@ -100,22 +101,30 @@ <h2 class="u-visually-hidden">
             {% if resource.icon %}
             <div class="media_image-container
                         media_image-container__wide-margin">
-                <img class="media_image u-centered-on-mobile"
-                     width="150"
+                <img class="media_image
```

Comment:
Scratch that, the icon field is just a URL to an image. I would imagine @sarahsimpson09 would just put in a URL to an uploaded media file which gets RAMPed with the page.